src/xunit.performance.api/PerformanceTestConfig.cs

```diff
@@ -0,0 +1,18 @@
+
+
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public class PerformanceTestConfig
+    {
+        public PerformanceTestConfig() 
+        {
+            iterations = 30;
+            timeout = 20000;
+        }
+
+        public int iterations;
```

Comment:
You should have some bounds checking to make sure that someone doesn't specify 0 or negative iteration count.